simulation/runSim.sh

```diff
@@ -0,0 +1,22 @@
+cd ~/coppelia
+SERVER="comm_server_scene.ttt"
+LIBRARY="libsimExtROSInterface.so"
+if [ -f $SERVER ]; then
+        echo "Server scene found."
+else
+        cd
+        SERVERPATH="$(find -name $SERVER | grep -m 1 $SERVER)"
+        cp $SERVERPATH coppelia/
+        cd coppelia
+        echo "Server scene not found. Moving..."
+fi
+if [ -f $LIBRARY ]; then
+        echo "ROS Library found."
+else
+        LIBPATH="$(find -name libsimExtROSInterface.so | grep -m 1 $LIBRARY)"
```

Comment:
```suggestion
        LIBPATH="$(find /home/duke/dev -name $LIBRARY | head -1)"
```
perhaps retrieving the file out of the tarball should be done in the dockerfile. the script should be used to retrieve it from the git repo if it is not found in the tarball (if it's definitely in the tarball then we can get rid of this section of the script).

Reply:
also, we could possible move runSim.sh to robosub-ros, there's no particular reason to have it int he docker image